### DISCUSSION TITLE
Support for adding tags to exif data

### DISCUSSION THREAD

**Discussion by felkami** [Simon-Tesla/RaccoonyWebEx#164] [Ideas]
Howdy! I'd like to be able to download images and have their tags and data written directly to the exif data of the image.

**Comment by Simon-Tesla** [comment]
Interesting idea, though currently images are downloaded using the browser's built-in downloading mechanism which doesn't grant me access to the raw data stream. Probably not impossible, but technically challenging from the POV of where the extension is at right now.
